containerized_lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}
```

Comment:
We should probably note that this variable exists and how to use it in the same doc